src/topics/resolver/github-resolver.ts

```diff
@@ -22,37 +29,69 @@ export class GithubResolver implements IResolver {
     };
   }
 
-  resolve = async (githubData: string): Promise<string> => {
-    const splitGithubData = githubData.split(":");
-    if (splitGithubData.length === 3) {
-      const id = githubData.split(":")[2];
-      const resolvedAccount = resolveAccount("1001", id);
-      return resolvedAccount;
-    }
+  resolve = async (githubData: FetchedData): Promise<FetchedData> => {
+    let githubDataUpdated = Object.entries(githubData).filter(
+      ([account, value]) => {
+        const splitGithubData = account.split(":");
+        if (splitGithubData.length === 3) {
+          const id = account.split(":")[2];
+          this.resolvedAccounts[resolveAccount("1001", id)] = value;
+        }
+        return splitGithubData.length !== 3;
+      }
+    );
+
+    githubDataUpdated = githubDataUpdated.map((data) => {
+      return [data[0].split(":")[1], data[1]];
```

Comment:
comment what you do quickly to be sure about the logic
You take the name and the value for the github account

Reply:
It allows me to transform this:
```
{
    'github:martingbz': '1',
    'github:leosayous21': '2',
    'github:yum0e': '3',
}
```
Into this:
```
[
    ['martingbz',1]
    ['github:leosayous21', 2]
    ['github:yum0e', 3]
]
```
Then I can resolve the account and find his associated value easily